src/resources/SearchInterfaces/SearchInterfaces.model.ts

```diff
@@ -0,0 +1,17 @@
+interface IAccesses {
```

Comment:
Do you mind adding jsdocs for each of those fields? See example [here](https://github.com/coveo/platform-client/blob/master/src/resources/Sources/SourcesMappings/SourcesMappingsInterfaces.ts).